src/inmanta/db/versions/v5.py

```diff
@@ -25,7 +25,7 @@ async def update(connection: Connection) -> None:
         """
 -- Compile queue might be collapsed if it contains similar compile requests.
 -- In that case, substitute_compile_id will reference the actually compiled request.
-ALTER TABLE public.compile ADD COLUMN substitute_compile_id uuid REFERENCES public.compile (id);
+ALTER TABLE public.compile ADD COLUMN substitute_compile_id uuid REFERENCES public.compile (id) ON DELETE CASCADE;
```

Comment:
v5 of the database schema was released last week, so this schema update should be part of v6.py.

Reply:
Is this still true if this is released as a patch? If so, should the next major release use v7?

Reply:
yes, otherwise is won't be applied to the things that have first deployed the release, then the patch